docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+services:
+  core:
+    container_name: coreservice
```

Comment:
You can remove this line, then update these below:

```yaml
STORE_SERVING_OPTIONS: '{"host": "redis", "port": "6379"}'
FEAST_CORE_HOST: core
```

With Docker Compose the service names are resolvable hostnames by default, not just the generated container names, so this avoids the `URISyntaxException` problem you were having with `feast_core_1` as a URL.